src/tiny_slam/tiny_scan_matcher.h

```diff
@@ -6,16 +6,32 @@
 #include "../core/monte_carlo_scan_matcher.h"
 #include "tiny_scan_cost_estimator.h"
 
+/*!
+ * \brief Derived class from MonteCarloScanMatcher to replace robot pose in space
+ *
+ * There is a logic how to choose a place of robot in the world (by generating vector of randomize variables - shifts of robot coordinates in the space);
```

Comment:
Consider smth like that "The simulation of based on pose movement by a random vector and rotation by a random angle. The vector distribution is dynamically adjusted."